server/src/main/java/io/crate/expression/eval/EvaluatingNormalizer.java

```diff
@@ -66,6 +66,7 @@
  *  - Convert {@link ScopedSymbol} to {@link Reference} if {@link FieldResolver} is available.
  *  - Convert {@link MatchPredicate} to a {@link Function} if {@link FieldResolver} is available
  *  - Convert {@link Reference} into a Literal value if {@link ReferenceResolver} is available
+ *  - Un-alias {@link AliasSymbol} to a {@link Reference}.
```

Comment:
```suggestion
 *  - Drops aliases ({@link AliasSymbol}) 
```

You're not guranteed to get a `Reference` beneath an alias, could also be a function, or scoped symbol